src/Maestro/Client/src/Generated/Builds.cs

```diff
@@ -407,7 +407,7 @@ public Builds(MaestroApi client)
 
             const string apiVersion = "2019-01-16";
 
-            var _path = "/api/builds/{id}/tree";
+            var _path = "/api/builds/{id}/graph";
```

Comment:
@alexperovich a few changes on this PR were automatically created when invoking the `generate-client.cmd' script but wasn't really my intention to create them. I'm not sure if I should leave them or not.

Reply:
These changes look correct. These are the api changes that are in prod.